app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt

```diff
@@ -70,25 +76,77 @@ class TransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDe
         viewModel.loadDetails(txId)
     }
 
-    private fun updateUi(txDetails: TransferDetails?, isLoading: Boolean) {
+    private fun updateUi(txDetails: TransactionDetails?, isLoading: Boolean) {
+
         binding.refresh.isRefreshing = isLoading
-        txDetails?.let {
-            binding.content.visible(true)
-            binding.txConfirmations.setExecutionData(
-                status = txDetails.txStatus,
-                confirmations = (txDetails.detailedExecutionInfo as? MultisigExecutionDetails)?.confirmations?.map { it.signer } ?: listOf(),
-                threshold = (txDetails.detailedExecutionInfo  as? MultisigExecutionDetails)?.confirmationsRequired ?: 0,
-                executor = txDetails.executor
-            )
-            binding.created.value = txDetails.createdAt?.formatBackendDate() ?: ""
-            binding.executed.value = txDetails.executedAt?.formatBackendDate() ?: ""
-            binding.etherscan.setOnClickListener {
-                requireContext().openUrl(
-                    getString(
-                        R.string.etherscan_transaction_url,
-                        txDetails.txHash
+        binding.content.visible(true)
+
+        
+        // TODO: How can we use viewBindings for the ViewStubs after they have been replaced?
```

Comment:
what about `TxDetailsTransferBinding.bind(view)`?

Also theoretically you only need one stub as you can change the target layout with setLayoutResource